tests/corobo_test_case.py

```diff
@@ -0,0 +1,36 @@
+from errbot.backends.test import FullStackTest
+from errbot.plugin_info import PluginInfo
+from errbot.templating import add_plugin_templates_path
+from pathlib import Path
+
+import logging
+
+
+class CoroboTestCase(FullStackTest):
+
+    def setUp(self, plug_filename):
+        super().setUp(loglevel=logging.ERROR,
+                      extra_config={'BACKEND': 'text'})
+        self.load_plug_file(plug_filename)
+
+    def load_plug_file(self, plug_filename):
```

Comment:
Since we are accepting plugin class, here's a way you can determine the filename from that. 
```py
filename = klass.__module__.split('.')[-1] + '.plug'
```

Reply:
Rename to `load_plugin_templates`